### PR TITLE
[Private IP Manager] Add UTs to Cover IpAddrRangeRepo interfaces

### PR DESCRIPTION
**Pull request by chenpiaoping**

### CODE REVIEW

File: services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        assertEquals(ipAddrAlloc.getState(), UnitTestConfig.activated);
+    }
+
+    @Test
+    public void allocateIpAddrWithoutIpAddrTest() throws Exception {
+        IpAddrRequest ipAddrRequest = buildIpAddrRequest();
+        ipAddrRequest.setIp(null);
+        IpAddrAlloc ipAddrAlloc = ipAddrRangeRepo.allocateIpAddr(ipAddrRequest);
+        assertNotNull(ipAddrAlloc);
+        assertEquals(ipAddrAlloc.getIpAddr(), UnitTestConfig.ip1);
```
Comment (xieus) [inline services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java:75]: This assertion shows that our algorithm is predictable :-)

File: services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        assertEquals(ipAddrAlloc.getState(), UnitTestConfig.activated);
+    }
+
+    @Test
+    public void allocateIpAddrWithoutRangeIdTest() throws Exception {
+        IpAddrRequest ipAddrRequest = buildIpAddrRequest();
+        ipAddrRequest.setRangeId(null);
```
Comment (xieus) [inline services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java:82]: Is the ip range selected deterministically or randomly, if more than more?

File: services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java (hunk 1 above)
Comment (chenpiaoping) [reply services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java:75]: yeah, sequentially.

File: services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java (hunk 2 above)
Comment (chenpiaoping) [reply services/private_ip_manager/src/test/java/com/futurewei/alcor/privateipmanager/repo/IpAddrTest.java:82]: sequentially

**Pull request merged**